module/metrics/transaction.go

```diff
@@ -97,6 +103,12 @@ func NewTransactionCollector(
 			Help:      "histogram for the duration in ms of the round trip time for executing a script",
 			Buckets:   []float64{1, 100, 500, 1000, 2000, 5000},
 		}, []string{"script_size"}),
+		scriptExecutionErrorOnExecutor: promauto.NewCounterVec(prometheus.CounterOpts{
+			Name:      "script_execution_error_archive",
+			Namespace: namespaceAccess,
+			Subsystem: subsystemTransactionSubmission,
+			Help:      "histogram for the internal errors for executing a script for a block on the archive node",
+		}, []string{"errors"}),
```

Comment:
`errors` is confusing
```suggestion
		}, []string{"source"}),
```